websites/W/WebtoonXYZ/dist/metadata.json

```
@@ -0,0 +1,42 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "service": "WebtoonXYZ",
+    "author": {
+      "name": "Harold",
+      "id": "141206144264699904"
+    },
+    "category": "anime",
+    "logo": "https://i.imgur.com/ZLoGNfR.png",
+    "thumbnail": "https://i.imgur.com/UkgngBO.png",
+    "version": "1.0.0",
+    "color": "#eb2e73",
+    "tags": [
+      "manhwa",
+      "webtoon"
+    ],
```

```suggestion
    "tags": [
      "manhwa",
      "webtoon",
      "nsfw"
    ],
```
forgot, this site includes a lot of NSFW movies and ads on homepage so add nsfw tag too

---

Waiting.